packages/orchestration/src/types.ts

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable no-use-before-define -- fine for types */
+// Ambients
+import '@agoric/zoe/exported.js';
+
+import type { Amount, Brand } from '@agoric/ertp/exported.js';
+import type {
+  CoinAmount,
+  CosmosChainInfo,
+  CosmosStakingFacet,
+  Denom,
+  IcaAccount,
+} from './cosmos-api.js';
+import type { EthChainInfo } from './ethereum-api.js';
+import type { ICQConnection } from './exos/icqConnectionKit.js';
+import type {
+  ChainAddress,
+  OrchestrationAccountI,
+  TransferMsg,
+} from './orchestration-api.js';
+
+export type * from './cosmos-api.js';
+export type * from './exos/chainAccountKit.js';
+export type * from './exos/icqConnectionKit.js';
+export type * from './orchestration-api.js';
+export type * from './service.js';
+export type * from './vat-orchestration.js';
+
+/**
+ * static declaration of known chain types will allow type support for
+ * additional chain-specific operations like `liquidStake`
+ */
+export type KnownChains = {
```

Comment:
This code ends up in vats, right? Let's not use typescript in vat code until we have a build process that does nothing more than removing types.

_Closest issue I can find is #5760._

Reply:
This module has no runtime. If it did, the bundle would fail because it sources from SCM and not the package dist

Reply:
oops. I misread. I missed the `type` in `export type ...`.

Reply:
TIL: `.d.ts` are taken as axioimatically correct by `tsc`.

@turadg explained that this is written in `.ts` so it gets checked.

_I wonder if that's worth a comment for other readers. hm._